probes/hasInProgressBadge/def.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: hasInProgressBadge
+short: This check determines whether the project has an OpenSSF (formerly CII) Best Practices Badge at the InProgress level.
```

Comment:
based on comment above this (and similar probes) may need modifications in this file if we take the "level is at least foo" approach

Reply:
This should no longer be relevant because of https://github.com/ossf/scorecard/pull/3520#discussion_r1406891624, so closing.